js/bitget.js

```diff
@@ -87,17 +87,19 @@ module.exports = class bitget extends Exchange {
             },
             'timeframes': {
                 '1m': '1m',
+                '3m': '3m',
                 '5m': '5m',
                 '15m': '15m',
                 '30m': '30m',
                 '1h': '1h',
+                '2h': '2h',
                 '4h': '4h',
-                '6h': '6h',
-                '12h': '12h',
-                '1d': '1d',
-                '3d': '3d',
-                '1w': '1w',
-                '1M': '1M',
+                '6h': '6Hutc',
+                '12h': '12Hutc',
+                '1d': '1Dutc',
+                '3d': '3Dutc',
+                '1w': '1Wutc',
+                '1M': '1Mutc',
```

Comment:
@ttodua I don't think this is the right fix, because we have timeframes split by market type inside `options` and that's what we currently use

Reply:
uh-oh, havenet noticed it. 

btw, I'll push the fix for this current PR in few mins, but then , in general, what do you think about having 2 timeframe block in describe? cant we somehow upgrade the approach, so we had one place for timeframe definitions ? It might be time to move , and have them  indexed by markettype  (inside the main `timeframes` property, and completely remove from `options`).